src/main/scala/org/ua/ilm/datastore/io/config/ConfigurationFile.scala

```diff
@@ -0,0 +1,8 @@
+package org.ua.ilm.datastore.io.config
+
+import com.typesafe.config.ConfigFactory
+
+class ConfigurationFile {
+  private val partFile = "application.conf"
```

Comment:
partFileName. and what about making it a class arg with default value "application.conf"